Projects/README.md

```diff
@@ -23,3 +23,4 @@ Each of the projects below are all self-funded and community-driven. If one (or
 * [Tesseract](Tesseract.md) - Open Crypto Equity and Assets Network
 * [Meridion](Meridion.md) - Open Project Governance-as-a-Service
 * [Castle in the Cloud](Castle-in-the-Cloud.md) - Easy DIY Sovereign Cloud Server
+* [LexiNomicon](Lexinomicon.md) - Nomicon gamespace for governance of Ulex bounties
```

Comment:
Where is Lexinomicon.md? We should be consistent with the spelling, maybe Lexinomicon (following the convention set by Cryptonomicon), not LexiNomicon. Or should we change to CryptoNomicon?

Reply:
Just added the Lexinomicon.md to the pull request.

Reply:
Still says LexiNomicon in the link.

Reply:
Fixed.